packages/cli/lib/format.js

```diff
@@ -0,0 +1,13 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+const prettier = require('prettier');
```

Comment:
_Cross-posting item 8 from https://github.com/strongloop/loopback-next/issues/441#issuecomment-392522724:_

Formatting of generated code: it would be great if our tooling was able to pick up prettier version and configuration from the project we are running in and format the generated code accordingly.

Imagine a user is using `prettier` in their project (as scaffolded by `lb4 app`), but changed prettier config to use double-quotes and tabs. If we don't honor their style, they will have to run `npm run lint:fix` every time they add a new model or update auto-generated TypeScript definitions.

It is also important to run `prettier` from `node_modules` of the target project, because different prettier version have variations in how they format the same piece of code and we cannot get all users to have exactly the same version of prettier as we use in LB4 tooling.

It may be enough to add `${this.destinationPath()}/node_modules` to the list of directories where Node.js looks for modules. See how Mocha handles that ([link](https://github.com/mochajs/mocha/blob/0d95e3f41a0859be4e7225541575e78b6d8926ee/bin/_mocha#L327)):

```ts
module.paths.push(cwd, join(cwd, 'node_modules'));
```

I am ok to leave these two improvements out of scope of this initial pull request as long as there is a follow-up issue (or two?) to ensure we fix this problem as part of the DP3 Boot epic.